unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarMeshCombiner/AvatarMeshCombinerHelper.cs

```diff
@@ -42,8 +42,13 @@ public class AvatarMeshCombinerHelper : IAvatarMeshCombinerHelper
         public bool enableCombinedMesh { get; set; } = true;
 
         private AvatarMeshCombiner.Output? lastOutput;
+        private bool keepPose = true;
```

Comment:
I would have this as a param of `CombineInternal` instead of a member of the class.